doc/man3/EVP_PKEY_CTX_set_params.pod

```diff
@@ -23,6 +23,8 @@ The EVP_PKEY_CTX_get_params() and EVP_PKEY_CTX_set_params() functions allow
 transfer of arbitrary key parameters to and from providers.
 Not all parameters may be supported by all providers.
 See L<OSSL_PROVIDER(3)> for more information on providers.
+The B<params> field is a pointer to a list of OSSL_PARAM structures, terminated
```

Comment:
Use `I<params>` and `B<OSSL_PARAM>` please.

Reply:
sure, fixed